src/containers/ReportArticle/index.js

```diff
@@ -0,0 +1,116 @@
+import React, { Component } from 'react';
+import Materialize from "materialize-css";
+import PropTypes from 'prop-types';
+import connect from "react-redux/es/connect/connect";
+import { bindActionCreators } from "redux";
+import Modal from "../../components/Modal";
+import DropDown from "../../components/DropDown";
+import DropDownItem from "../../components/DropDownItem";
+import { fetchViolationTypesAction, reportArticleAction } from "./state/actions";
+
+
+class ReportArticle extends Component {
+  componentDidMount = () => {
+    Materialize.Modal.init(document.querySelector('#report-modal'), {});
+
+    const { fetchViolationTypes } = this.props;
+    fetchViolationTypes();
+  };
+
+  componentDidUpdate = () => {
+    Materialize.FormSelect.init(document.querySelectorAll('select'), {});
+  };
+
+  state = {
+    type: "",
+    description: "",
+  };
+
+  onChange = (e) => {
+    this.setState({ [e.target.name]: e.target.value });
+  };
+
+  handleSubmit = (event) => {
+    event.preventDefault();
+    const { article, history } = this.props;
+    const data = { ...this.state };
+    this.props.reportArticle(article.slug, data, () => {
+      history.push("/");
```

Comment:
@imireallan @mutaimwiti @gitaumoses4 ,
Which page do you think I should redirect to once the user reports an article?